carbonmark/components/pages/Project/Retire/Pool/RetireInputs.tsx

```diff
@@ -175,21 +275,29 @@ export const RetireInputs: FC<Props> = (props) => {
               </div>
             )}
             control={control}
-            options={Object.values(carbonmarkPaymentMethodMap).map((val) => ({
-              id: val.id,
-              label: val.label,
-              value: val.id,
-              icon: val.icon,
-              disabled: val.disabled,
-            }))}
+            options={Object.values(carbonmarkRetirePaymentMethodMap).map(
+              (val) => ({
+                id: val.id,
+                label: val.label,
+                value: val.id,
+                icon: val.icon,
+                disabled: val.disabled,
+              })
+            )}
           />
         </div>
 
+        {exceededFiatBalance && (
+          <Text t="body1" className={styles.errorMessagePrice}>
+            {getValidations().totalPrice.max.message}
+          </Text>
+        )}
+
         <div className={styles.paymentHelp}>
           <HelpOutline className={styles.helpIcon} />
           <div className={styles.paymentText}>
             <Text t="body3">
-              {t`To retire this project using a different form of payment,`}{" "}
+              {t`Currently, Carbonmark only accepts Polygon USDC or Credit Card Payments. To retire this project using a different form of payment,`}{" "}
               <Anchor href={urls.app}>click here</Anchor>.
```

Comment:
Complete string should be in the tag for translators. also "click here" is bad a11y practice, lets change the text:
Edit: I've asked for thumbs up from product team to remove this help text altogether. I don't think its needed.
```suggestion
              <Trans>Currently, Carbonmark only accepts Polygon USDC or Credit Card Payments. To retire this project using a different form of payment, <Anchor href={urls.app}>visit app.klimadao.finance</Anchor>.</Trans>
```

Reply:
I changed the text now with the correct blog links etc according to the Figma designs.

We can remove everything later again when the feedback from product came back.